shared/constant/constant.go

```diff
@@ -38,6 +43,41 @@ const (
 )
 
 var TrustedCloneImages = []string{
```

Comment:
we should also lint and create a warning so users will be notifyed if the clone step image is not trusted ...

Reply:
5dfed307f6a2708552ae11018029c013d7dbd7a6